src/control/server/harness_test.go

```diff
@@ -578,6 +563,79 @@ func TestServer_Harness_CallDrpc(t *testing.T) {
 			expErr:         FaultDataPlaneNotStarted,
 			expFailHandler: true,
 		},
+		"none available; not leader": {
+			notLeader: true,
+			mics: []*MockInstanceConfig{
+				{
+					Ready:       atm.NewBool(true),
+					CallDrpcErr: errDRPCNotReady,
+				},
+				{
+					Ready:       atm.NewBool(true),
+					CallDrpcErr: FaultDataPlaneNotStarted,
+				},
+			},
+			expNotLeader:   true,
+			expErr:         FaultDataPlaneNotStarted,
+			expFailHandler: true,
+		},
+		"none available; no drpc related errors": {
+			mics: []*MockInstanceConfig{
+				{
+					Started:     atm.NewBool(true),
+					CallDrpcErr: errEngineNotReady,
+				},
+				{
+					Ready:       atm.NewBool(true),
+					CallDrpcErr: errors.New("whoops"),
+				},
+			},
+			expErr: errors.New("whoops"),
+		},
```

Comment:
Is this equivalent to the case removed above?

Reply:
yes, covers the same case